x-pack/plugins/apm/server/lib/errors/distribution/get_buckets.ts

```diff
@@ -27,15 +28,7 @@ export async function getBuckets({
   const filter: ESFilter[] = [
     { term: { [PROCESSOR_EVENT]: 'error' } },
     { term: { [SERVICE_NAME]: serviceName } },
-    {
-      range: {
-        '@timestamp': {
-          gte: start,
-          lte: end,
-          format: 'epoch_millis'
-        }
-      }
-    }
+    { range: rangeFilter(start, end) }
```

Comment:
Replaced with `rangeFilter` helper. Note that this only works for apm data with `@timestamp` (not the @)